model/version.go

```diff
@@ -1,5 +1,6 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Did we intend to change this one from 2015 to 2020?

Reply:
just checked we need to use `2015-present` will update all